x-pack/plugins/cloud_security_posture/server/types.ts

```diff
@@ -29,3 +36,23 @@ export interface CspServerPluginStartDeps {
   data: DataPluginStart;
   fleet: FleetStartContract;
 }
+
+export type CspRequestHandlerContext = FleetRequestHandlerContext;
```

Comment:
Since the CSP plugin already depends on the Fleet plugin,  you already have access to Fleet's request handler context in route handlers, however these types are needed to be able to make TypeScript happy.